uaclient/tests/test_util.py

```diff
@@ -68,6 +68,39 @@
 """
 
 
+class TestIsContainer:
```

Comment:
I think this needs some negative tests; something like:

```py
def is_container(run_path=None):
    try:
        util.subp(['systemd-detect-virt', '--quiet', '--container'])
    except:
        pass
    return True
```

would pass this testing, and is obviously incorrect. :-)

Reply:
good point. fixed and pushed